build.sbt

```diff
@@ -10,13 +10,18 @@ val allScalaVersions = scala2Versions :+ scala3Version
 ThisBuild / scalaVersion       := scala213
 ThisBuild / crossScalaVersions := allScalaVersions
 
-addCommandAlias("ci-test", "scalafmtCheckAll; scalafmtSbtCheck; mdoc; ++test")
+addCommandAlias(
+  "ci-test",
+  "scalafmtCheckAll; scalafmtSbtCheck; documentation/mdoc; ++test; ++github4sJS/test"
```

Comment:
If I'm not wrong, `++test` will execute the task for all the modules

Reply:
I did not know this notation.
Then if we the output twice in the ci, we can remove `++github4sJS/test`

Reply:
`++test` will run the test for all the supported scala versions plus scalajs. no need to add `++github4sJS/test` :)